renovate.json

```diff
@@ -1,6 +1,36 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+"$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base"
-  ]
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "packageRules": [
+    {
+      "description": "Tag the waddlers Github Team for major updates",
+      "matchUpdateTypes": [
+        "major"
+      ],
+      "reviewers": [
+        "team:waddlers"
```

Comment:
Do waddlers own this code? Is there anyone else we could add maybe?

Reply:
Bad copy/paste. I mean, no. But I'm not sure anyone owns it, so I think that means Infra owns it. :lolsob: